src/views/officer.j2

```diff
@@ -1,10 +1,10 @@
 {% for field in metadata.fields %}
-{% set value = record.get(field.FieldName) %}
+{% set value = record.get(field.FieldName) | string %}
```

Comment:
Oh, smart.

Reply:
Yea I was having trouble with the `if not boolean` so I just opted for this lol